src/scss/_components.diagram.chart.scss

```diff
@@ -75,48 +111,156 @@ $circle-height: 28px;
   }
 
   &__rectangle {
-    --width: 4rem;
-
     position: absolute;
-    left: calc(50% - (var(--width) * 0.5));
+    left: calc(50% - 4rem * 0.5);
```

Comment:
Since this is a value that is being used multiple times, make this a Sass variable.

Reply:
I don't see it being reused in the current state, but if it ends up being repeated it definitely makes sense to make a variable then!